packages/broker/src/config/migration.ts

```diff
@@ -144,7 +145,7 @@ export const readConfigAndMigrateIfNeeded = (fileName: string | undefined): Conf
     let explicitTargetFile = undefined
     if (fileName === undefined) {
         const defaultTargetFile = getDefaultFile()
-        const legacyTargetFile = getLegacyDefaultFile()
+        const legacyTargetFile = path.join(os.homedir(), '.streamr/broker-config.json')
```

Comment:
It used to be used once, here, and assigned to `legacyTargetFile`. Does not seem to _need_ a utility function per se, no?

Reply:
Nothing _needs_ a utility function... But we put that into a utility function so that config-related facts are in `config.ts`. If a function which gives us the default file is there, then a function that does almost the same could be also in the same place.

Maybe the actual solution would be to drop the support for legacy file location. The legacy location was used in back in Brubeck, so shouldn't be relevant in Tatum era 👍

Reply:
The util is back.